breathecode/certificate/views.py

```diff
@@ -25,9 +26,39 @@
 
 @api_view(["GET"])
 @permission_classes([AllowAny])
-def get_specialties(request):
-    items = Specialty.objects.all()
-    serializer = SpecialtySerializer(items, many=True)
+@capable_of("read_certificate")
+def get_academy_specialties(request, academy_id=None):
+
+    # Filter by the old syllabus (OneToOneField) and the new syllabuses (ManyToManyField)
+    items = Specialty.objects.filter(
+        Q(syllabus__academy_owner=academy_id) | Q(syllabuses__academy_owner=academy_id)
```

Comment:
`syllabuses__academy_owner` being syllabuses a list, can I access the elements inside the list to filter by them with `syllabuses__academy`? @jefer94

Reply:
Remove the filter by syllabus, keeping only the syllabuses lookup @gustavomm19